utils/answer-types.js

```diff
@@ -1216,6 +1216,7 @@ Khan.answerTypes = $.extend(Khan.answerTypes, {
                         // otherwise correct or not before forwarding on the
                         // message.
```

Comment:
Seems to be still valid (the unsimplified fraction is the only case that the branch is taken that I'm aware of).

If a student enters "2/4" into a proper fraction field a "your answer is almost correct" message is normally displayed. (Under the default mode of simplification.) Such an answer does not fall under any of the three usual categories of "empty", "correct", or "incorrect", thus the framework uses a special score for it -- the score is marked as "empty", but carries the message.

Now the multiple exercise has a surprising semantic: if any subscore is "empty", but not all are, then the combined score is incorrect (even if all nonempty parts are actually correct; this happens as empty scores have `correct: false`). Without the branch with the lengthy comment, any multiple with an unsimplified part and another correct part would be graded wrong.

If we were to let multiple be empty whenever at least one of its parts is empty and all nonempty parts are correct, then this special case would be unnecessary -- a single unsimplified fraction would cause the combined score to be empty. And an empty score is treated as unanswered, displaying the message it carries. That's the content of https://github.com/wrwrwr/khan-exercises/commit/9a25e476baa21ed4e43a33ef3b6d44dd3e9d44a0 (not part of this PR).